integration_tests/test_list_01.py

```diff
@@ -18,4 +18,20 @@ def test_list():
     for i in range(15):
         assert (f[i] - a[i]) == (i + 1.0)
 
-test_list()
+def test_list_02():
+    x: list[f64] = [1.0, 2.0]
+    y: list[f64] = []
+    i: f64
+
+    for i in x:
+        y.append(i)
+
+    j: i32
+    for j in range(len(y)):
```

Comment:
`len(y)` works in your branch?

Reply:
Nah, Not yet. We need to wait until the #930 is merged.